core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/OxfordNanoporeContainer.java

```diff
@@ -0,0 +1,66 @@
+package uk.ac.bbsrc.tgac.miso.core.data.impl;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+@Entity
+@Table(name = "OxfordNanoporeContainer")
+public class OxfordNanoporeContainer extends SequencerPartitionContainerImpl {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2160)